fe/model_rabfe.py

```diff
@@ -559,7 +564,20 @@ def setup_topology(self, mol_a, mol_b):
 
 class AbsoluteConversionModel(AbsoluteModel):
     def setup_topology(self, mol):
-        top = topology.BaseTopologyConversion(mol, self.ff)
+        with open(os.path.join(os.path.dirname(ff_path), "params/smirnoff_1_1_0_ccc.py")) as f:
+            ff_handlers = deserialize_handlers(f.read())
+
+        ref_forcefield = Forcefield(ff_handlers)
+
+        ff_params = ref_forcefield.get_ordered_params()
+
+        def ff_conversion(mol: Chem.Mol) -> NDArray:
```

Comment:
@maxentile This might need to be jnp rather than np.

If you have any suggestions on how to do this in a cleaner way let me know.

Reply:
I think the `np.concatenate` needs to be `jnp.concatenate` (contains an array conversion, which jax won't like when tracing), but otherwise looks fine!